staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -374,6 +374,17 @@ func LogLibvirtLogLine(logger *FilteredLogger, line string) {
 	pos := strings.TrimSpace(fragments[3])
 	msg := strings.TrimSpace(fragments[4])
 
+	//TODO: implement proper behavior for unsupported GA commands
+	// by either considering the GA version as unsupported or just don't
+	// send commands which not supported
+	if strings.Contains(msg, "unable to execute QEMU agent command") {
+		if logger.verbosityLevel >= 4 {
```

Comment:
How about

```go
if logger.verbosityLevel < 4 {
  return
}
severity = LogLevelNames[WARNING]

Reply:
@rmohr Yeah feels like more readable. Done

Reply:
> /approve
> 
> @enp0s3 did you verify that the message goes away?
> 

@rmohr Wroking on it now